src/ui-test/uitest_runner.ts

```diff
@@ -38,7 +41,10 @@ async function main(): Promise<void> {
 			'settings': './src/ui-test/resources/vscode-settings.json',
 			resources: []
 		});
-	fs.rmSync(extensionFolder, { recursive: true });
 }
 
 main();
+
+function cleanup() {
+	fs.rmSync(variables.launcher.EXTENSION_DIR, { recursive: true, force: true });
+}
```

Comment:
good habit to add an empty line at the end. it eases future code review in case more lines of code ar added to the class